Assets/Scripts/Ability System/AttackBuffAbility.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+[CreateAssetMenu]
+public class AttackBuffAbility : Ability
+{
+    public float damageMultiplier;
+
+    public override void Activate(GameObject parent)
+    {
+        Debug.Log("attack buff start");
+        Debug.Log("Original attack: " + parent.GetComponent<Player>().Attack);
+        parent.GetComponent<Player>().Attack *= damageMultiplier;
```

Comment:
Use the ModifiableStats system. Convert attack from a float on the player to a ModifiableStat. Then, make an AttackBuff status effect. When the ability is active, apply the status effect to the player. You'll also need to add enums for the attackStat (modifiablestat) and for the damage multiplier (buff). If you have questions you can ping me @xD0G3x#3030 on Discord

I didn't think of the use case of easily applying a single buff with the modifiable stats system. I'll add the ability to add single modifiers more easily in my next PR